src/main/java/com/endava/soa_5/array_and_string/ArraySum.java

```diff
@@ -0,0 +1,39 @@
+package com.endava.soa_5.array_and_string;
+
+public class ArraySum {
+    ArraySum() {
+    }
+
+    //this method create array (lengthOfArray) length, and value equal with own indexes
+    public static int[] createArrayWithValues(int lengthOfArray) {
+        int[] tempArray = new int[lengthOfArray];
```

Comment:
Sounds very ambiguous to me "tempArray" temp = temperature or temporary? Please give a more explicit name for this variable.

Reply:
Done!